docs/Kubernetes/kubernetes.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+Note: Not for production use!!
+This guide with provide detail on how to deploy the AMTCloudToolkit services in Microsoft(C) Azure(R).
+Functionality has been added to MPS to allow it to scale and support a greater number of devices. 
+For this deployment, kubernetes will be used running in Azure(R) along with utilizing both redis and consul.
+
+### High level Design
+
+####Figure #1 MPS Scaling Architecture
```

Comment:
"Figure 1: MPS Scaling Architecture"
(just to be consistent with the rest of the diagrams, using a colon)

"Figure 1 presents the high-level architecture of MPS scaling mode. Per the figure, starting at the bottom:"

(The only one i changed here is number 2.)
1. Devices connect to an available MPS Server through the load balancer.
2. "The REST API requests are routed to an available Web Server. a component of MPS running in scale mode, though a load balancer."
3. The Web Server determines which MPS Server to route the traffic to based on which MPS Server the device is connected to and sends that traffic through the MPS Proxy connection. 
4.. The MPS Server sends the traffic to the corresponding device.